File: src/main/java/no/nav/foreldrepenger/stønadskonto/regelmodell/regler/Kontokonfigurasjon.java
package no.nav.foreldrepenger.stønadskonto.regelmodell.regler;

import no.nav.foreldrepenger.stønadskonto.regelmodell.StønadskontoKontotype;
import no.nav.foreldrepenger.stønadskonto.regelmodell.konfig.Parametertype;

record Kontokonfigurasjon(StønadskontoKontotype stønadskontotype, Parametertype parametertype) {
}

File: stonadskonto/src/regler/Kontokonfigurasjon.ts
import { StønadskontoKontotype } from '../StønadskontoKontotype';
import { Parametertype } from '../konfig/Parametertype';

export interface Kontokonfigurasjon {
  stønadskontotype: StønadskontoKontotype;
  parametertype: Parametertype | null;
}

export function createKontokonfigurasjon(
  stønadskontotype: StønadskontoKontotype,
  parametertype: Parametertype | null
): Kontokonfigurasjon {
  return { stønadskontotype, parametertype };
}

